### PR TITLE
Coffee minigame

### PR DESCRIPTION
**Pull request by wouterz**
New pullrequest to release branch. 

This PR contains a minigame for the janitor, Coffee run. For now its just numbers on the buttons, might add images later as an enhancement.

Pay extra attention to the networking please

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/PixelPerfect.iml" filepath="$PROJECT_DIR$/PixelPerfect.iml" />
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/pixelperfect-client.iml" filepath="$PROJECT_DIR$/pixelperfect-client.iml" />
```
Comment (jessetilro) [inline .idea/modules.xml]: Why is this line added? It believe it breaks my project configuration. (It isn't even valid since there is no such module.)

Comment (wouterz) [inline .idea/modules.xml]: same as 2 lines above, except my project is localy named  'pixelperfect-client' instead of 'PixelPerfect'. Dont think this breaks it, but can remove it if needed.

Comment (jessetilro) [inline .idea/modules.xml]: My IDE cannot locate this module since it has not created it, leaving the project "out of sync".

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/CoffeeBoostActivity.java [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+import nl.tudelft.pixelperfect.pixelperfect.R;
+import nl.tudelft.pixelperfect.pixelperfect.Spaceship;
+
+/**
+ * Mini-game for the Janitor.
+ *
+ * @author Wouter Zirkzee
+ *
+ */
+public class CoffeeBoostActivity extends Activity {
```
Comment (jessetilro) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/CoffeeBoostActivity.java:26]: This looks great. Maybe later you could create a separate component for the logic of the puzzle, instead of putting it all in the Activity?

**Comment by jessetilro** [comment]
Tested manually, works great. Networking also looks fine! Approved.

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/CoffeeBoostActivity.java [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    /**
+     * Check if the game is complete and send a message.
+     *
+     * @param view
+     *          A view.
+     */
+    public void confirmCoffeeEvent(View view) {
+        if (gameComplete()) {
+            game.sendMessage(new EventCompletedMessage(EventTypes.COFFEE_BOOST.ordinal()));
+        }
```
Comment (jessetilro) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/CoffeeBoostActivity.java:180]: Send the user back to the Janitor Activity after pressing the button. This way the player cannot remain the solution and spam ik whenever necessary. When failing the player must try all over again.

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/CoffeeBoostActivity.java (hunk 2 above)
Comment (wouterz) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/CoffeeBoostActivity.java:26]: Might do this later, but for now i think its fine all in one class

**Comment by FlXlT** [comment]
Please discard every misc.xml or modules.xml changes you come across.

**Comment by FlXlT** [comment]
Manual tested it, mini-game looks good.
